core/readme/.prettierrc

```diff
@@ -1,4 +1,5 @@
 {
+  "endOfLine": "auto",
```

Comment:
We need this also in the main repo.

Reply:
I think that when we refactor our eslint config to be more modular and unify there the rules, prettier will contains this for all, included main repo

Reply:
Agreed